src/software/ai/hl/stp/tactic/crease_defender_tactic.cpp

```diff
@@ -125,8 +125,8 @@ std::optional<std::pair<Point, Angle>> CreaseDefenderTactic::calculateDesiredSta
 
         // Find the best shot
         auto best_shot =
-            calcBestShotOnFriendlyGoal(field, friendly_team, enemy_team, ball.position(),
-                                       ROBOT_MAX_RADIUS_METERS, {robot});
+            calcBestShotOnGoal(field, friendly_team, enemy_team, ball.position(),
+                               TeamType::FRIENDLY, ROBOT_MAX_RADIUS_METERS, {robot});
```

Comment:
for example, this would simplify to: 
```suggestion
            calcBestShotOnGoal(field, friendly_team, enemy_team, ball.position(),
                               TeamType::FRIENDLY, {robot});
```